test/Distributed/Inputs/FakeDistributedActorSystems.swift

```diff
@@ -300,7 +303,9 @@ public struct FakeInvocationEncoder : DistributedTargetInvocationEncoder {
 }
 
 // === decoding --------------------------------------------------------------
-public class FakeInvocationDecoder : DistributedTargetInvocationDecoder {
+
+@available(SwiftStdlib 5.7, *)
+public final class FakeInvocationDecoder: DistributedTargetInvocationDecoder {
```

Comment:
How hard would it be to add enum and struct variants for the decoder so all possibilities are tested? I think we could gyb this and some other tests too?

Reply:
I'll make a radar for it but right now I think that's a bit of a pain... I have no idea about gyb tbh

Reply:
As soon as we are keeping track of that I am fine, I just don’t want it to be broken without us noticing…

Reply:
Right yeah 👍

Reply:
rdar://91735574